includes/plugins/woocommerce-subscriptions/class-subscriptions-tiers.php

```diff
@@ -114,7 +114,12 @@ public static function print_switch_subscription_modal() {
 				continue;
 			}
 			$product = wc_get_product( reset( $parent_products ) );
-			self::render_modal( $product, null, null, $switch_data );
+			$label = __( 'Change subscription', 'newspack-plugin' );
+			if ( Donations::is_donation_product( $product->get_id() ) ) {
+				$title = __( 'Edit donation', 'newspack-plugin' );
+				$label = __( 'Confirm donation', 'newspack-plugin' );
```

Comment:
Non-blocking: is it possible to change the label of this button too, based on the same logic?

<img width="491" height="259" alt="Screenshot 2025-10-16 at 2 23 25 PM" src="https://github.com/user-attachments/assets/cdf9e315-4f58-461c-bfd0-b4079f8af90c" />